### DISCUSSION TITLE
RemoveInfo does not include title

### DISCUSSION THREAD

**Discussion by madebyankur** [mdn/mdn-community#618] [Content] [answered]
I'm encountering an issue where `removeInfo.node.title` returns undefined and isn't available in the `removeInfo` object either.

This article specifically shows an example with this in use though: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/onRemoved

May I ask what the issue might be? Is this expected behavior?

**Comment by wbamberg** [comment, marked as answer]
It would be better to ask this question in https://discourse.mozilla.org/c/add-ons/development/108.

**Reply by madebyankur** [reply]
Sure. There's an unanswered discussion around this from 2018, so I'll probably check that.

Thanks!

**Comment by AnilSeervi** [comment]
`removeInfo.node` is only available for versions of chrome above 48.
You can check the API here - https://developer.chrome.com/docs/extensions/reference/bookmarks/#event-onRemoved
Also you'll only be access it if you have set it beforehand while creating the bookmark.

Note that safari doesn't support the API.

**Comment by pepelsbey** [comment]
Closing as answered/settled.
